source/_posts/2016-03-23-rake-db-migrate-vs-rake-db-schema-load.markdown

```diff
@@ -0,0 +1,106 @@
+---
+layout: post
+title: "rake db:schema:load vs rake db:migrate"
+date: 2016-03-23 08:09:25 +0100
+comments: true
+author: Ivica Lakatoš
+categories:
+  - Ruby and Rails
+tags:
+  - ruby
+  - rails
+  - rake task
+  - globalize
+published: true
+---
+
+Sooner or later every new ruby developer need to understand  differences between this two common rake task. Basically, simple definition tell us everything we need to know:
```

Comment:
Just a few grammar issues I noticed, it should be: 'every new ruby developer need**s**', 'this two common rake task**s**' and '**a** simple definition tell**s** us'